CHANGELOG.md

```diff
@@ -77,7 +77,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Fixed handling of escaped characters in .gitignore (only keep trailing spaces
   if escaped properly).
 
-* `jj undo` now works after `jj duplicate`.
+* `jj duplicate` followed by `jj rebase` of a tree containing both the original
+  and duplicate commit no longer crashes. Also, `jj undo` now works after
+  `jj duplicate`.
```

Comment:
The fix is general enough to fix #27, right? If so, add that to the commit message (or PR), and make the this changelog entry more generic.

Hooray, btw! :)

Reply:
I don't understand that bug very well. I'm sure you are right, will look into it and put it here. 🎉

Reply:
I think that's about right. I'm not sure if there are any instances of #27 left apart from #694 but, if so, they should be resolved.